plugins/link/plugin.js

```diff
@@ -49,20 +49,25 @@
 
```

Comment:
It doesn't seem to be part of the fix for #654. If #654 does not make sense without #678, it should be blocked on it.

What's more, if #678 is resolved, these changes will become obsolete.